gpii/node_modules/eventLog/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "eventLog",
+    "description": "Event logging.",
+    "version": "0.3.0",
```

Comment:
That's what every/most other modules are.. I just copied it from somewhere.

And, it's more stable than 0.1.